samples/SocketsSample/wwwroot/index.html

```diff
@@ -24,6 +24,12 @@
         <li><a href="hubs.html?transport=ServerSentEvents&hubType=dynamic">Server Sent Events</a></li>
         <li><a href="hubs.html?transport=WebSockets&hubType=dynamic">Web Sockets</a></li>
     </ul>
+    <h1>ASP.NET Core SignalR (Hub&ltT>)</h1>
```

Comment:
nit: encode the greater-than too, it's good practice (but technically not necessary).